modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/linear-segregated/2d/2d-boussinesq.i

```diff
@@ -0,0 +1,243 @@
+mu = 2.6
+rho = 1.0
+advected_interp_method = 'average'
+cp = 300
+k = 20
+alpha_b = 1e-4
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '1.5'
+    dy = '0.3'
+    ix = '55'
+    iy = '20'
```

Comment:
This is just for regression, the real tests will go to the flow test bed. I think this should be a 10x5 little channel which is fast to run and cheap to store (the issue with SIMPLE that the number of cells increase the number of iterations it needs for convergence which is not nice for regression cases which typically need tighter tolerance). You can use `git rebase -i HEAD~number` to modify the gold file instead of regolding here.

Reply:
modified